pytest_django/fixtures.py

```diff
@@ -140,11 +142,16 @@ def _django_db_fixture_helper(
         from django.test import TransactionTestCase as django_case
 
         if reset_sequences:
-
             class ResetSequenceTestCase(django_case):
                 reset_sequences = True
 
             django_case = ResetSequenceTestCase
+
+        if serialized_rollback:
+            class SerializedRollbackTestCase(django_case):
```

Comment:
Creating another subclass seems unnecessary. I will combine with the above (can call it `PytestDjangoTestCase`).